core/node/da_clients/Cargo.toml

```diff
@@ -55,3 +55,14 @@ pbjson-types.workspace = true
 
 # Eigen dependencies
 tokio-stream.workspace = true
+rand.workspace = true
+sha3.workspace = true
+tiny-keccak.workspace = true
+ethabi.workspace = true
+rust-kzg-bn254.workspace = true
+ark-bn254.workspace = true
+num-bigint.workspace = true
+serial_test.workspace = true
```

Comment:
This looks like a dev dependency.

Reply:
fixed here https://github.com/lambdaclass/zksync-era/pull/375